packages/@ourworldindata/grapher/src/mapCharts/MapChart.tsx

```diff
@@ -884,6 +889,9 @@ class ChoroplethMap extends React.Component<{ manager: ChoroplethMapManager }> {
 
         const clipPath = makeClipPath(uid, bounds)
 
+        // this needs to be referenced here or it will be recomputed on every mousemove
+        const _cachedCentroids = this.quadtree
```

Comment:
The `.quadtree` accessor is marked as `@computed` but doesn't actually persist across callbacks. Referencing it here gives the proper memoization, but I can't say I understand why (or what a less cryptic way of achieving that end would look like)…